tiny_dnn/io/caffe/layer_factory_impl.h

```diff
@@ -660,7 +674,7 @@ inline std::shared_ptr<layer> create_convlayer(
   }
 
   auto conv = std::make_shared<conv_layer>(
-    in_width, in_height, window_size, in_channels, out_channels, table,
+    in_width, in_height, window_size_h, window_size_w, in_channels, out_channels, table,
```

Comment:
The order is reversed. https://github.com/tiny-dnn/tiny-dnn/blob/master/tiny_dnn/layers/convolutional_layer.h#L116_L117